clash-prelude/src/Clash/Tutorial.hs

```diff
@@ -1187,9 +1187,9 @@ BlackBox:
       begin
         if ~IF~ACTIVEEDGE[Rising][0]~THENrising_edge~ELSEfalling_edge~FI(~ARG[3]) then
           if ~ARG[7] ~IF ~ISACTIVEENABLE[4] ~THEN and ~ARG[4] ~ELSE ~FI then
-            ~SYM[2](~SYM[5]) <= ~TOBV[~ARG[9]][~TYP[9]];
+            ~SYM\[2](~SYM[5]) <= ~TOBV[~ARG[9]][~TYP[9]];
```

Comment:
Instead of these very precise escapes in problem areas, have considered changing to the bird track style code blocks?

Ie change the whole block from:
```
@
line 1
...
line n
@
```
to:
```
> line 1
> ...
> line n
```
As I understand it those don't have any parsing formatting/special characters.

You can't do that for some of the code blocks where you want to have embedded links for functions used. But when you don't need that, like in these blackboxes, it'll be more robust against future edits.

Reply:
I 'unno. It's a large diff, it's busy work, I find the `@...@` style more pleasant to look at, and it doesn't solve a fundamental issue. But if you feel strongly, I can change it.

BTW, I did check the whole source for more instances of this issue. And there aren't any more instances. Your grep correctly identified all of them.

Reply:
>  it'll be more robust against future edits.

I feel somewhat defeated in this area. It feels like putting out a single twig in a forest fire. Haddock is just really hard to write and maintain.

Reply:
> I feel somewhat defeated in this area. It feels like putting out a single twig in a forest fire. Haddock is just really hard to write and maintain.

:(